src/applications/edu-benefits/0993/containers/SubmitButtons.jsx

```diff
@@ -123,7 +110,7 @@ export default function SubmitButtons(props) {
   }
   return (
     <div>
-      <div className="row form-progress-buttons">
+      <div className="row form-progress-buttons schemaform-buttons">
```

Comment:
This class, which is typically wrapped around the ProgressButtons in the us-forms-system's FormPage component:
```
          <div className="row form-progress-buttons schemaform-buttons">
            <div className="small-6 medium-5 columns">
              <ProgressButton
                onButtonClick={this.goBack}
                buttonText="Back"
                buttonClass="usa-button-secondary"
                beforeText="«"/>
            </div>
            <div className="small-6 medium-5 end columns">
              <ProgressButton
                submitButton
                buttonText="Continue"
                buttonClass="usa-button-primary"
                afterText="»"/>
            </div>
          </div>
```
 ensures the submit button is aligned, otherwise the result is the following:
![image](https://user-images.githubusercontent.com/16051172/42616098-c8ba7be4-8561-11e8-9cd3-775783e0a982.png)